browser/ui/BUILD.gn

```diff
@@ -785,6 +790,29 @@ source_set("ui") {
       "//components/permissions",
     ]
   }
+  if (is_android) {
+    sources += [
+      "webui/brave_wallet/android/swap_page_ui.cc",
+      "webui/brave_wallet/android/swap_page_ui.h",
+      "webui/brave_wallet/common_handler/wallet_handler.cc",
+      "webui/brave_wallet/common_handler/wallet_handler.h",
+      "webui/brave_wallet/page_handler/wallet_page_handler.cc",
+      "webui/brave_wallet/page_handler/wallet_page_handler.h",
+      "webui/brave_wallet/wallet_common_ui.cc",
+      "webui/brave_wallet/wallet_common_ui.h",
+      "webui/brave_wallet/wallet_page_ui.cc",
+      "webui/brave_wallet/wallet_page_ui.h",
+    ]
+    deps += [
+      "//brave/browser/ui/brave_wallet",
```

Comment:
do we need to include the whole wallet ui? Is it because of `wallet_bubble_manager_delegate.h`. I don't think we need that bubble delegate on Android.

Reply:
fixed